app/drivers/manager.go

```diff
@@ -833,12 +894,24 @@ func (m *Manager) setupInstance(
 		createOptions.Platform = pool.Platform
 	}
 
+	if isCapacityTask {
```

Comment:
I was talking about this here - https://github.com/drone-runners/drone-runner-aws/pull/688/files#r2470641919.
if it's a capacity task then you just create a capacity reservation and no instance creation in database. When will the cloud init run for this case ?